pkgs/os-specific/bsd/freebsd/pkgs/mkDerivation.nix

```diff
@@ -89,7 +89,7 @@ lib.makeOverridable (
       # TODO should CC wrapper set this?
       CPP = "${stdenv'.cc.targetPrefix}cpp";
 
-      # Since STRIP below is the flag
+      # Since STRIP in `makeFlags` is has to be a flag, not the binary itself
```

Comment:
wait hang on. this is some really bad grammar lol

remove the `is`

Reply:
oops!